Dockerfile

```
@@ -1,10 +1,24 @@
-FROM golang:1.12
+FROM golang:1.12-alpine as build
 
 WORKDIR /src
```

It's generally good practice to create the workdir manually, instead of relying on the behaviour of `WORKDIR` creating the directory. in this case, we're running as `root` (so not a big difference), but if the image / build is executed as non-root, then manually creating the directory makes sure it gets the right permissions/ownership;

```suggestion
RUN mkdir -p /src
WORKDIR /src
```

I'm also wondering if the default go path should be used here;

```suggestion
RUN mkdir -p /go/src/github.com/whalebrew/whalebrew/
WORKDIR /go/src/github.com/whalebrew/whalebrew/
```

---

> I'm also wondering if the default go path should be used here;
Because we are using modules, and that modules, from go 1.11, are enabled outside of ${GOPATH}. This though means that `go install` can be dropped its prefix `GO111MODULE=on`